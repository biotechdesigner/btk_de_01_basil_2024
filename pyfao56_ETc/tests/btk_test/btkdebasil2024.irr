************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Irrigation Data
Timestamp: 04/18/2024 11:00:41
************************************************************************
Comments: 2024 basil
************************************************************************
Year-DOY  Depth     fw
2024-102   0.00   1.00
2024-103   0.00   1.00
2024-104   0.00   1.00
2024-105   0.00   1.00
2024-106   0.00   1.00
2024-107   0.00   1.00
2024-108   0.00   1.00