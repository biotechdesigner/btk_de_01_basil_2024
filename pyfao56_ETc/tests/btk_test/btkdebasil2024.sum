************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Seasonal Water Balance Summary
Timestamp: 04/19/2024 08:29:45
Simulation start date: 04/11/2024
Simulation end date: 04/18/2024
All values expressed in mm.
************************************************************************
Comments: 2024 basil previous test
************************************************************************
  18.266 : ETref
   7.306 : ETc
   0.000 : ETcadj
   0.000 : E
   0.000 : T
   0.000 : DP
   0.000 : Irrig
   0.000 : Rain
   4.000 : Dr_ini
   4.000 : Dr_end
 -99.999 : Drmax_ini
 -99.999 : Drmax_end
