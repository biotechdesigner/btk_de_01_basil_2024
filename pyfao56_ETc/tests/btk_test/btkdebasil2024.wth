************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Weather Data
Timestamp: 04/02/2024 11:01:47
************************************************************************
Comments:
************************************************************************
           S Reference crop - Short ('S') or Tall ('T')
 389.0000000 Weather station elevation (z) (m)
  48.7111700 Weather station latitude (decimal degrees)
   2.0000000 Wind speed measurement height (m)

Daily weather data:
Year-DOY   Srad   Tmax   Tmin   Vapr   Tdew  RHmax  RHmin  Wndsp   Rain  ETref   MorP
2024-077    NaN  13.30   4.30    NaN    NaN  98.90  58.80   0.50   0.00   1.28      M
2024-078    NaN  11.20   7.20    NaN    NaN  98.70  89.20   0.50   0.00   0.55      M
2024-079    NaN  15.20   2.30    NaN    NaN 100.00  60.30   0.50   0.00   1.55      M
2024-080    NaN  18.30   3.60    NaN    NaN  98.30  42.30   0.50   0.00    NaN      M
2024-081    NaN  15.50   6.80    NaN    NaN  90.20  63.30   0.50   0.00    NaN      M
2024-082    NaN  18.40   4.90    NaN    NaN  96.00  45.70   0.50   0.00    NaN      M
2024-083    NaN   9.90   2.00    NaN    NaN  91.80  65.20   0.50   0.00    NaN      M
2024-084    NaN   6.00   1.90    NaN    NaN  93.40  70.40   0.50   0.00    NaN      M
2024-085    NaN  11.10   2.70    NaN    NaN  89.80  39.10   0.50   0.00    NaN      M
2024-086    NaN  17.10   1.70    NaN    NaN  86.50  28.60   0.50   0.00    NaN      M
2024-087    NaN  12.50   4.70    NaN    NaN  93.10  64.50   0.50   0.00    NaN      M
2024-088    NaN  10.80   0.90    NaN    NaN  93.20  50.80   0.50   0.00    NaN      M
2024-089    NaN  15.50   4.40    NaN    NaN  91.60  61.60   0.50   0.00    NaN      M
2024-090    NaN  16.20   6.50    NaN    NaN  91.70  40.90   0.50   0.00    NaN      M
2024-091    NaN  18.90   6.20    NaN    NaN  95.80  46.30   0.50   0.00    NaN      M
2024-092    NaN  11.60   7.10    NaN    NaN  95.80  68.40   0.50   0.00    NaN      M