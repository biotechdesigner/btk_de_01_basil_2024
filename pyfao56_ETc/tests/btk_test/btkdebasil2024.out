************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Output Data
Timestamp: 04/18/2024 11:00:41
Simulation start date: 04/11/2024
Simulation end date: 04/17/2024
Soil method: D - Default FAO-56 homogenous soil bucket approach
************************************************************************
Comments: 2024 basil previous test
************************************************************************
Year-DOY  Year  DOY  DOW      Date  ETref  tKcb   Kcb     h Kcmax    fc    fw   few      De    Kr    Ke      E     DPe    Kc    ETc     TAW TAWrmax    TAWb    Zr     p     RAW    Ks Kcadj ETcadj      T      DP    Dinc      Dr     fDr   Drmax  fDrmax      Db     fDb   Irrig    Rain  Year  DOY  DOW      Date
2024-102  2024  102  Thu  04/11/24  2.695 0.400 0.400 0.100 1.200 0.000 1.000 1.000  12.573 0.000 0.000  0.000   0.000 0.400  1.078   4.000 -99.999 -99.999 0.050 0.557   2.228 0.000 0.000  0.000  0.000   0.000 -99.999   4.000   1.000 -99.999 -99.999 -99.999 -99.999   0.000   0.000  2024  102  Thu  04/11/24
2024-103  2024  103  Fri  04/12/24  2.954 0.400 0.400 0.100 1.193 0.000 1.000 1.000  12.573 0.000 0.000  0.000   0.000 0.400  1.182   4.000 -99.999 -99.999 0.050 0.553   2.211 0.000 0.000  0.000  0.000   0.000 -99.999   4.000   1.000 -99.999 -99.999 -99.999 -99.999   0.000   0.000  2024  103  Fri  04/12/24
2024-104  2024  104  Sat  04/13/24  3.373 0.400 0.400 0.100 1.209 0.000 1.000 1.000  12.573 0.000 0.000  0.000   0.000 0.400  1.349   4.000 -99.999 -99.999 0.050 0.546   2.184 0.000 0.000  0.000  0.000   0.000 -99.999   4.000   1.000 -99.999 -99.999 -99.999 -99.999   0.000   0.000  2024  104  Sat  04/13/24
2024-105  2024  105  Sun  04/14/24  2.965 0.400 0.400 0.100 1.185 0.000 1.000 1.000  12.573 0.000 0.000  0.000   0.000 0.400  1.186   4.000 -99.999 -99.999 0.050 0.553   2.210 0.000 0.000  0.000  0.000   0.000 -99.999   4.000   1.000 -99.999 -99.999 -99.999 -99.999   0.000   0.000  2024  105  Sun  04/14/24
2024-106  2024  106  Mon  04/15/24  1.645 0.400 0.400 0.100 1.187 0.000 1.000 1.000  10.973 0.000 0.000  0.000   0.000 0.400  0.658   4.000 -99.999 -99.999 0.050 0.574   2.295 0.000 0.000  0.000  0.000   0.000 -99.999   2.400   0.600 -99.999 -99.999 -99.999 -99.999   0.000   1.600  2024  106  Mon  04/15/24
2024-107  2024  107  Tue  04/16/24  1.973 0.400 0.400 0.100 1.168 0.000 1.000 1.000  10.903 0.350 0.269  0.530   0.000 0.669  1.319   4.000 -99.999 -99.999 0.050 0.547   2.189 0.945 0.647  1.276  0.746   0.000 -99.999   3.076   0.769 -99.999 -99.999 -99.999 -99.999   0.000   0.600  2024  107  Tue  04/16/24
2024-108  2024  108  Wed  04/17/24  1.344 0.400 0.400 0.100 1.154 0.000 1.000 1.000   0.973 0.365 0.275  0.370   0.000 0.675  0.907   4.000 -99.999 -99.999 0.050 0.564   2.255 0.705 0.557  0.749  0.379   6.475 -99.999   0.000   0.000 -99.999 -99.999 -99.999 -99.999   0.000  10.300  2024  108  Wed  04/17/24